### ISSUE TITLE
Create Settings Tab/Page

### ISSUE THREAD

**Issue by JRhooooo** [jotrujil03/bird-dropper#11]
As a user, I want to be able to access options regarding my profile and website through a settings tab for easier access to accessibility options. I should be able to access options to change my username, password, email. etc.
